### PR TITLE
Remove format

### PR DESCRIPTION
**Pull request by marcofalcioni**

### CODE REVIEW

**Comment by marcofalcioni** [comment]
@ekofman this is the real commit.

File: python/ClinicalReportLaunchers/launch_panel_report_new_genome.py [hunk 1]
```diff
@@ -122,24 +122,40 @@ def upload_genome_to_project(project_id, label, sex, file_format, file_name):
 
 
 def main(argv):
-    """Main function, uploads a geneome and creates a panel report using it.
+    """Main function, uploads a genome and creates a panel report using it.
     """
+<<<<<<< HEAD
```
Comment (UnquietCode) [inline python/ClinicalReportLaunchers/launch_panel_report_new_genome.py]: @marcofalcioni Seems like an unfinished merge happening here.

File: python/family_folder [hunk 2]
```diff
@@ -0,0 +1 @@
+../../omicia_api/api_smoke_test/family_folder
```
Comment (UnquietCode) [inline python/family_folder:1]: gut check: Is it safe to use symbolic links in our git repositories? Probably, but I'm not 100% sure.

Comment (marcofalcioni) [reply python/family_folder:1]: the api smoke test needs to be packaged better. right now it only runs from Eric's laptop (or mine, maybe). This is a manifestation of the problem.

**Comment by ekofman** [comment]
cool looks good. And yes agreed, at some point I need to take some time to make the api smoke tests more generalizable and usable by anyone.

**Pull request closed without merging**